src/main/java/prefs/Preferences.java

```diff
@@ -118,6 +118,16 @@ public void addBoard(String name, List<PanelInfo> panels) {
     public void removeBoard(String name) {
         global.removeBoard(name);
     }
+
+    public void setLastDeletedIndex(String deletedBoardName){
```

Comment:
@damithc Do I still need to add header comments if i refactor it to `setLastDeletedBoardIndex`?